web-ui/src/pages/ReviewsPage.jsx

```diff
@@ -1,43 +1,46 @@
-import React, {useCallback, useEffect, useState} from "react";
-import { styled } from '@mui/material/styles';
-import { useLocation, useHistory } from 'react-router-dom';
+import React, {useCallback, useEffect, useState} from 'react';
+import {styled} from '@mui/material/styles';
+import {useLocation, useHistory} from 'react-router-dom';
 import queryString from 'query-string';
-import ReviewPeriods from "../components/reviews/periods/ReviewPeriods";
-import TeamReviews from "../components/reviews/TeamReviews";
+import ReviewPeriods from '../components/reviews/periods/ReviewPeriods';
+import TeamReviews from '../components/reviews/TeamReviews';
 
 const PREFIX = 'ReviewPage';
 const classes = {
   root: `${PREFIX}-root`,
   headerContainer: `${PREFIX}-headerContainer`,
   requestHeader: `${PREFIX}-requestHeader`,
-  stepContainer: `${PREFIX}-stepContainer`,
+  stepContainer: `${PREFIX}-stepContainer`
 };
 
 const Root = styled('div')(({theme}) => ({
   [`&.${classes.root}`]: {
-    backgroundColor: "transparent",
-    margin: "4rem 2rem 2rem 2rem",
-    height: "100%",
-    'max-width': "100%",
```

Comment:
I changed all CSS properties that do not use camelCase names to use them in order to remove warnings.

Reply:
Good call.